components/CatalogSearch.vue

```diff
@@ -119,11 +133,15 @@
         }
     }
 
+    .icon.pill__close {
+        font-size: map-get($font-size-phone, 5);
+    }
```

Comment:
Please use the class p5 instead of defining it here. Add the class p5 to this element, by doing so the font-size is scaling responsive.